src/shared/util.js

```diff
@@ -591,13 +591,9 @@ function string32(value) {
                              (value >> 8) & 0xff, value & 0xff);
 }
 
+// NOTE: This name is misleading as it rounds the result up
```

Comment:
Does this match the previous behavior? If not, why do we round up in the new version?